pkg/asset/manifests/azure/cluster.go

```diff
@@ -240,3 +253,32 @@ func GenerateClusterAssets(installConfig *installconfig.InstallConfig, clusterID
 		},
 	}, nil
 }
+
+func getNextAvailableIP(ctx context.Context, installConfig *installconfig.InstallConfig) (string, error) {
+	lbip := "10.0.0.100"
```

Comment:
DefaultInternalLBIPAddress is an exported value: 
https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/cc3e8e91c9fbcc4e2199eb0de3cd01466869c844/api/v1beta1/azurecluster_default.go#L43
So we could use that instead of the magic string

```suggestion
	lbip := "capz.DefaultInternalLBIPAddress"
```